src/containers/Tenant/Diagnostics/Overview/utils/prepareChangefeedInfo.ts

```diff
@@ -0,0 +1,34 @@
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+
+import {formatObject, InfoViewerItem} from '../../../../../components/InfoViewer';
+import {
+    formatCdcStreamItem,
+    formatCommonItem,
+} from '../../../../../components/InfoViewer/formatters';
+
+import {prepareTopicSchemaInfo} from './prepareTopicSchemaInfo';
+
+export const prepareChangefeedInfo = (
```

Comment:
this was in CDCStreamInfo.tsx, you moved it in utils, correct?
Do you expect this to be a shared function that would be required in other components? If yes, ok, if not — why do we need utils for this?